src/govuk-prototype-kit/govuk-prototype-kit.config.mjs

```diff
@@ -0,0 +1,37 @@
+import slash from 'slash'
```

Comment:
Ah-ah!
Separator of the universe
Slash
Ah-ah!
He'll separate out the filesystem!

(That was in my head and I wanted to share it with you)